web/app/view/MapController.js

```diff
@@ -166,6 +182,43 @@ Ext.define('Traccar.view.MapController', {
                 if (marker === this.selectedMarker && this.followSelected()) {
                     this.getView().getMapView().setCenter(marker.getGeometry().getCoordinates());
                 }
+
+                if (deviceId in this.liveRoutes) {
+                    lastLiveCoordinates = this.liveRoutes[deviceId][this.liveRoutes[deviceId].length - 1].getGeometry().getCoordinates();
+                    if (lastLiveCoordinates[0] === geometry.getCoordinates()[0] &&
```

Comment:
This `if` prevent tails cuts every 30 seconds in debug mode because of websocket re-connection. Also it is useful if device send positions (with same coordinates) while parking.
Only thing is `jscs` complains about `continue` statement.

What should I do?
- Remove workaround
- Refactor code to avoid `continue`
- Configure `jscs` to not complain about `continue`